client/src/components/Public/GameForm.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+  TextField,
+  Button
+} from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  form: {
+    '& .MuiTextField-root': {
+      margin: theme.spacing(1),
+      width: '100%',
+    },
+  },
+  public: {
+    display: 'block',
+    margin: 'auto',
+    marginTop: theme.spacing(1),
+    width: "50%",
+    fontWeight: "bold",
+  },
+  title: {
+    textAlign: 'center'
+  }
+}));
+
+const Btn = ({ on, css, text }) => (
+  <Button onClick={on} className={css} variant="outlined">{ text }</Button>
+);
+
+const GameForm = ({ onCreateGame, playerName, setPlayerName }) => {
+  const classes = useStyles()
+  const [gameRoomName, setGameRoomName] = useState('player1\'s game');
+  const [playerNum, setPlayerNum] = useState(4);
+
+  const selectItems = [];
+  for (let i = 4; i <= 8; i++) {
```

Comment:
Realistically this can probably just be 4,6,8 max players

Reply:
Since every guesser in a team plays at the same time during their turn and shares same amount of guess chances, so I thought it is not important to force each team to have same number of players. I don't know do we want to leave it as is or make changes for it.